### PR TITLE
Tls eve log fix v1.0

### PR DESCRIPTION
**Pull request by regit**
This patchset contains one patch introducing a new 'tls-store' module and a second restoring backward compatibility.

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/1150

PR builds:
- PR regit: https://buildbot.openinfosecfoundation.org/builders/regit/builds/86
- PR regit-pcap: https://buildbot.openinfosecfoundation.org/builders/regit-pcap/builds/84

### CODE REVIEW

File: src/log-tlsstore.c [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+/**
+ *  \internal
+ *
+ *  \brief deinit the log ctx and write out the waldo
+ *
+ *  \param output_ctx output context to deinit
+ */
+static void LogTlsStoreLogDeInitCtx(OutputCtx *output_ctx)
+{
+    free(output_ctx);
```
Comment (inliniac) [inline src/log-tlsstore.c]: SCFree?

File: src/log-tlsstore.c [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+    SCLogWarning(SC_ERR_FWRITE, "Unable to write certificate");
+end_fwrite_fpmeta:
+    if (fpmeta) {
+        fclose(fpmeta);
+        SCLogWarning(SC_ERR_FWRITE, "Unable to write certificate metafile");
+    }
+    SCReturn;
+end_fp:
+    fclose(fp);
+}
```
Comment (inliniac) [inline src/log-tlsstore.c]: minor: missing return, while we have it everywhere else in the func

**Comment by inliniac** [comment]
Right, so ultimately this just adds another flag to the state for keeping the eve cert logging and this modules logging independent, correct?

**Comment by regit** [comment]
Yes, exactly.

**Comment by regit** [comment]
Going to propose a updated PR.

**Comment by inliniac** [comment]
btw I noticed that we log errors that we could encounter at packet runtime, meaning we could be flooded with messages. This isn't new in the pr, but still something to consider maybe?

**Pull request closed without merging**